__tests__/integration/backend-proxy.test.js

```diff
@@ -140,5 +150,18 @@ describe('Backend proxy integration', () => {
 				expect(response.headers.location).toEqual(relativePath);
 			});
 	});
+
+	test('intercepts an error from the backend', () => {
+		const scope = nock(backend).get('/interceptOn')
+			.reply(401, 'You are not authorised to view this content', {'content-type': 'text/plain'});
+
+		return request.get('/interceptOn')
+			.then(response => {
+				scope.done();
+
+				expect(response.status).toBe(200);
```

Comment:
Should this status code be the same as what the backend returned?

Reply:
Interestingly no, that would be up to the project's error handler. Though maybe it would be useful to forward that status code on for the tests.

Reply:
So how would the FE know what the BE error is, so it can e.g. template a 500 and 404 differently?

Reply:
You would do something with the status code. Typically you'd have an error handling middleware like

```
app.use((error, req, res, next) => {
   res.status(error.statusCode);
   switch (error.statusCode) {
        case 404: return res.render('404', error);
        ...
  }
});
```

Regarding the status code, I'm honestly not sure. Perhaps it's best for the `backendProxy` to set the `response.status` itself so the correct status code does get passed to the client.

Reply:
I think that might be an idea. I think I would expect the response status to be that of the backend, and switch on that, rather than introspecting the error (if I am understanding you correctly!)

Reply:
At some point you will have to do something with the `error`, but (as of the latest changes) we now set the `response.statusCode` to match the `backendResponse` so even if you don't render different templates the client will get the correct error code